docs/features/opencdc-record.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: 'OpenCDC record'
+sidebar_position: 2
+---
+
+An OpenCDC record in Conduit aims to standardize the format of data records exchanged between different connectors within a data processing pipeline. The primary objective is to ensure compatibility between various combinations of source and destination connectors.
+
+## Benefits
+
+1. **Support for Operations**: The format should support representing records for create, update, delete, and snapshot operations.
+2. **Standard Metadata Fields**: Definition of standard metadata fields to provide essential information about each record. These can vary depending on the record. See [Metadata Constants](#metadata-constants) for more information.
+3. **Integration with Data Tools**: We believe that being strict about the record format Conduit consumes and produces will make it easier to integrate with other data processing tools.
+
+## Fields
```

Comment:
Can we add a `:::note` callout at the end of this chapter explaining why we are writing `.Position` and not `.position`? Basically explain that we are using the [Go templates](https://pkg.go.dev/text/template) notation for the Go representation of the record (we could even link to it), and because the fields are public they start with an uppercase letter.